### DISCUSSION TITLE
add a mod to be able to use Shaders

### DISCUSSION THREAD

**Discussion by ZombieKiller9549** [KdGaming0/SkyBlock-Enhanced-Modpack#10] [Ideas]
i found this modpack off of a hypixel forum [https://hypixel.net/threads/mod-help.5642455/](url) and noticed there wasn't a place for me to enable shaders. if you could add that or suggest a mod that would fit this modpack that would be nice!

**Comment by KdGaming0** [comment]
Hello, and thanks for taking an interest in my modpack. To use shaders you have to download Optifine (this is recommended even if you don't use shaders because it gives a big boost in fps). On the Modpacks Modrinth page there is a link to a video showing how to install it and a written guide if you want to use it instead. 
Because of the license, Optfine uses can I not have it as a part of the modpack and you will have to download it manually.
![image](https://github.com/user-attachments/assets/72f06852-5dd1-4cb8-9fc7-d801040ff3ad)

**Reply by ZombieKiller9549** [reply]
i see, thank you for responding in a fast manner. when i get around to it, i shall install optifine (i know how to install it from previous modded encounters). I hope you have an excellent christmas (if you celebrate it 😊)
